war/src/main/js/upgradeWizard.js

```diff
@@ -1,40 +1,38 @@
-onSetupWizardInitialized(function(wizard) {
```

Comment:
need to add a hint indicating this is a global method. it's something like `/*global onSetupWizardInitialized*/` see: http://stackoverflow.com/a/7866920